app/handlers/uploadHomography.py

```diff
@@ -1,23 +1,38 @@
 #!/usr/bin/env python
 
 import tornado.web
+import os
+from trafficcloud.app_config import get_project_path
```

Comment:
Change the import on all handlers to use the app_config found in traffic_cloud_utils folder